main.py

```diff
@@ -1,4 +1,4 @@
-import asyncio, websockets, sys, click, time, os
+import asyncio, websockets, sys, click, urllib3, time
```

Comment:
`urllib3` is a not-so elegant module but lightweight - something which is perfect for our usecase. Thanks for replacing `requests` in favour of this module.